src/index.ts

```diff
@@ -48,8 +53,15 @@ export class FlashbotsBundleProvider extends providers.JsonRpcProvider {
     this.genericProvider = genericProvider;
   }
 
-  async sendRawBundle(signedBundledTransactions: Array<string>, targetBlockNumber: number): Promise<FlashbotsTransactionResponse> {
-    await this.send("eth_sendBundle", [signedBundledTransactions, `0x${targetBlockNumber.toString(16)}`]);
+  async sendRawBundle(signedBundledTransactions: Array<string>, targetBlockNumber: number, opts?: FlashbotsOptions): Promise<FlashbotsTransactionResponse> {
+    const params = [`0x${targetBlockNumber.toString(16)}`] as Array<string | number>
+    if (opts?.minTimestamp) {
```

Comment:
The FlashBotsOptions object is optional, but if you do send it along, you must provide the entire thing. This parsing makes it seem like you could pass a maxTimestamp without a minTimestamp, which the type system as configured doesn't not allow

Reply:
If the optionals were setup this way, submitting a maxTimestamp without a minTimestamp, it would be interpreted as a minTimestamp

Reply:
ah good catch, @jparyani is it true that right now you can submit one or the other ?